lib/logflare/backends/adaptor/postgres/repo/migrations/add_log_events.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Logflare.Backends.Adaptor.Postgres.Repo.Migrations.AddLogEvents do
+  use Ecto.Migration
+
+  def up do
+    create table(:log_events, primary_key: false) do
+      add(:id, :string, primary_key: true)
+      add(:metadata, :map)
+      add(:event_message, :string)
+      add(:timestamp, :utc_datetime_usec)
+    end
```

Comment:
basic ingestion should only create 3 columns, id, event_message, timestamp.
metadata should be removed.

Reply:
do we want basic ingestion? the focus of this feature for now is to provide self service without BQ and for that effect we need metadata to replicate the queries we have in studio correct?

Reply:
Minimally would need to be able to support top level keys, namely the `project` key for project ref.

Reply:
we now store the full body but also have event_message, timestamp and id as separate columns to be closer to log_events in BQ